### ISSUE TITLE
587ae109f7916d16a0d06687559a2015f2d9e75e breaks login

### ISSUE THREAD

**Issue by csutcliff** [joelpob/betfairng#11]
Hi, the change of URL in the commit above does the opposite for me, it breaks login but changing it back works fine.

**Comment by chrisw000** [comment]
I just tried it again... having -api on the url still gives me the 400, but without I can log on.

.. not sure what difference it would make, but I'm in the UK and currently using a delay app key. The .p12 file and certificates etc. are in date etc.

**Comment by csutcliff** [comment]
Strange, I'm also in the UK but using a non-delay app key. Cert etc is all fine. Just that one change breaks/fixes it for me.

**Comment by chrisw000** [comment]
Have you got a delay key you could try - maybe that is an issue? Code would be simple enough to have it be told if it's a delay/live key or not and then user the appropriate URL

**Comment by csutcliff** [comment]
I will give it a go and get back to you

**Comment by joelpob** [comment]
I've only got a live key, and I'm based in Australia. 

Happy to take a pull request if you figure out that live/delay requires a switch.

**Comment by csutcliff** [comment]
Had time to do a little bit of mucking around. I'm actually getting a "CERT_AUTH_REQUIRED" response with the new URL. Works fine with the old one. Cert is exactly the same both times. I've noticed that the cert is actually out of date so I'll create a new one and attach it to the account to see if that's the issue.

**Comment by csutcliff** [comment]
Ok so this was a weird one!

I found that both end-points worked with the example console application, with old (expired) cert and with the new one I generated. Oddly only the old certificate worked with the old URL.

After a bit of head scratching I broke out wireshark and found that the only discernible difference between my application and the console example was that mine appeared to be using TLS 1.2 instead of TLS 1.0.

Adding "ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls;" to the force the TLS version used in my app means that both URLs now work!

Apparently TLS 1.2 is now the default for .net 4.6.x which is why I wasn't seeing this in the console example (.net 4.5).

TL;DR; Default TLS version changed in .net 4.6, Betfair doesn't like it!

**Comment by joelpob** [comment]
Nice, good catch!
